crates/sui-indexer/src/test_utils.rs

```diff
@@ -119,6 +161,31 @@ pub async fn start_test_indexer_v2(
     (store, handle)
 }
 
+fn replace_db_name(db_url: &str, new_db_name: &str) -> (String, String) {
+    let pos = db_url.rfind('/').expect("Unable to find / in db_url");
+    let old_db_name = &db_url[pos + 1..];
+
+    (
+        format!("{}/{}", &db_url[..pos], new_db_name),
+        old_db_name.to_string(),
+    )
+}
+
+pub async fn force_delete_database(db_url: String) {
+    // Connect to the `postgres` database to drop the active database.
+    // This is necessary because you can't drop a database while being connected to it.
+
+    // Replace the database name with `postgres`, which should be the last string after `/`
+    let (default_db_url, db_name) = replace_db_name(&db_url, "postgres");
```

Comment:
Why do we need to do this step?

Reply:
Cannot drop a db while connected to it. Need to connect to another db first.